features/google/src/model/importers/utils/namedOffsets.json

```diff
@@ -0,0 +1,292 @@
+{
```

Comment:
nit: should we mention here in a comment the link of the resource of this list for future reference?

Reply:
Yeah, I think that is a good idea, but json does not support comments.

Reply:
Also, this does not look like something we should have in a specific feature. If there's no library that supports it (really weird, but OK), it might be better handled at the silly-i18n level.

Reply:
I am not sure this should be part of an i18n package. It is more date related, not i18n related, since these are not internationalized offsets. They are just offsets with a name, and any date in any language can have them.

Reply:
For example an english date written as 23rd of July 1995 16:00 EEST, is a perfectly valid 😄

Reply:
And a romanian date 23 Iulie 1994 16:00 EEST is again valid

Reply:
Sorry, you lost me here. I've tried to add EEST to a date in Safari, and it does not work. But others, like CEST or PST do work. So is it an issue with specific timezones?

Reply:
Agreed, this has no business being in _silly-i18n_ -> We're parsing Google data here, so the Google feature is the right place. If we need it for multiple importers, we could add it to _poly-import_.